[13.0][FIX] account_payment_order: Fix tests

joao-p-marques opened the pull request:
On multi-company settings, we need to make sure we are searching and creating objects with the correct company associated.
(Includes part of https://github.com/OCA/bank-payment/pull/786)

Create account and bank journal instead of searching.
Add proper expected taxes to that account to allow predicting the final value.

@Tecnativa
TT28423
TT28962

ping @victoralmau @pedrobaeza

victoralmau commented on account_payment_order/tests/test_payment_order_inbound.py:
```diff
@@ -14,25 +14,30 @@ class TestPaymentOrderInboundBase(SavepointCase):
-        self.journal = self.env["account.journal"].search(
-            [("type", "=", "bank")], limit=1
+        self.journal = self.env["account.journal"].create(
+            {"name": "Test journal", "type": "bank"}
```
```suggestion
            {"name": "Test journal", "code": "BANK", "type": "bank"}
```

joao-p-marques replied:
Ok, change made, but why is this necessary? I see that it automatically fills that field of not defined.

joao-p-marques committed:
[FIX] account_payment_order: Fix tests

On multi-company settings, we need to make sure we are searching and creating objects with the correct company associated.
(Includes part of https://github.com/OCA/bank-payment/pull/786)

TT28423

Create account and bank journal instead of searching.
Add proper expected taxes to that account to allow predicting the final value.

TT28962

pedrobaeza commented:
/ocabot merge patch
